### ISSUE TITLE
IE11 Array.from on List from immutable.js

### ISSUE THREAD

**Issue by dmitin-bv** [Financial-Times/polyfill-service#1434]
I have React application with Redux-form.
Looks like immutable.js is used internally by React (or Redux).
Originally I was going to report an issue for immutable.js,
but I realized that issue is reproducible only when polyfill is used (under IE11).
Application works fine under FireFox (without polyfill).

Under IE11 (with polyfill) next line in parseIterableLike() looks to be a problem
```
var arr = parseIterable(usingIterator(items));
```

That line is called from Array.from polyfill.
In my case it should convert List from immutable.js to Array, but in fact original List is returned.
As result "items" parameter is assigned to arrayLike as is (without conversion to Array), because in parseIterableLike() at the end "arr" is false.
```
var arrayLike = Object(parseIterableLike(items));
```

Because of no conversion in line below I receive 0, since arrayLike.length is undefined (List from immutable.js has only correct size property)
```
var len = toInteger(arrayLike.length);
```

Even after manual changing "len" to correct size of list (using debugger), at the end I receive array of "undefined" items, since in next line "[]" operator doesn't work (because arrayLike is not an array)
```
kValue = arrayLike[k];
```

**Comment by dmitin-bv** [comment]
Well, I successfully applied next workaround
```
<script src="https://cdn.polyfill.io/v2/polyfill.min.js?features=es6|gated,es7|gated"></script>
<script type="application/javascript">
  (function () {
    'use strict';

    var tmp = Array.from;

    // The length property of the from method is 1.
    Object.defineProperty(Array, 'from', {
      configurable: true,
      value: function from(arrayLike/*, mapFn, thisArg */) {
        if (arrayLike._tail === undefined || arrayLike._tail === null) {
          return tmp.apply(this, arguments);
        } else {
          return arrayLike._tail.array.slice()
        }
      },
      writable: true
    });
  }());
</script>
```

**Comment by dmitin-bv** [comment]
I believe root cause is some issue with Symbol.iterator. We have another bug in our code because of this.

**Comment by JakeChampion** [comment]
Yes, I have rebuilt these polyfills to work correctly with custom iterables, the updates should go out in  version 3.26.

**Comment by jennifergarcia** [comment]
@JakeChampion do you know approximately when this will be released to QA and to production?

**Comment by JakeChampion** [comment]
3.25 is currently on QA and should be in production next week. This means that 3.26 will go to QA at some point this month (I am aiming for 20th Jan) and production by end of the month.

**Comment by jennifergarcia** [comment]
@JakeChampion it looks like 3.26 hasn't been released yet, do you have an update on when it's going to be tagged and released?

**Comment by jennifergarcia** [comment]
@JakeChampion, Is this going to be released anytime soon? We have several issues we are waiting to resolve related to this.

**Comment by jennifergarcia** [comment]
@JakeChampion do you know when this is going to be released?
